aptos-move/e2e-tests/src/executor.rs

```diff
@@ -519,7 +519,7 @@ impl FakeExecutor {
 
         let onchain_config = BlockExecutorConfigFromOnchain {
             // TODO fetch values from state?
-            block_gas_limit_type: BlockGasLimitType::Limit(30000),
+            block_gas_limit_type: BlockGasLimitType::Limit(1000000000),
```

Comment:
can we make this a const maybe?

Reply:
it is going to become a utility method on BlockExecutorConfigFromOnchain in #10943